pkg/util/validation/validation.go

```diff
@@ -139,6 +139,24 @@ func IsDNS1035Label(value string) []string {
 	return errs
 }
 
+// wildcard definition - RFC 1034 section 4.3.3.
+const wildcardDNF1123SubdomainFmt = "\\*\\." + dns1123SubdomainFmt
```

Comment:
Can you clarify this regex with some examples in the comment? 
According to https://tools.ietf.org/html/rfc2818#section-3.1,  "f*.com matches foo.com but not bar.com.", but this regex will fail "f*.com" right? Also https://tools.ietf.org/html/rfc6125#section-6.4.3 states that only the leftmost label will tolerate a "*", which I believe this regex handles.